python/example_robot_data/robots_loader.py

```diff
@@ -531,6 +575,10 @@ def loadIris():
 }
 
 
+if version.parse(pin.__version__) >= version.parse("2.9.2"):
+    ROBOTS['cassie']= CassieLoader
```

Comment:
I don't see the need for the conditional

Reply:
I was tottering on whether to include a version check or not, but I agree it doesn't add much value

Reply:
I've removed the commit